_releases/make-release-page.py

```diff
@@ -261,12 +261,12 @@ def binaryNameToTitle(filename):
 
         return F"macOS {macOSVers} {arch} Xcode {xcodeVers}"
     elif platformCompiler.startswith('win32') or platformCompiler.startswith('win64'):
-        regexWin = re.compile(r'^win(32|64)[.](vc[0-9]+)(.debug)?')
+        regexWin = re.compile(r'^win(32|64)[.](python3[0-9]+)[.](vc[0-9]+)(.debug)?')
```

Comment:
We don't build `debug` anymore. Should we distribute `relwithdebinfo`? Or do we distribute only `release` binaries?

Reply:
So far we have distributed release binaries: this can be changed if we believe that relwithdebinfo is useful to users. The regex is at the moment harmless, in the sense that in case debug binaries are available, the markdown of the page is generated accordingly. To be 100% correct, one should indeed detect the last part of the binary name and attach it to the name accordingly: release by default, debug in case we have debug binaries and release with debug info (see above necessity criterion)